front_end/html/view_answer.html

```diff
@@ -0,0 +1,25 @@
+<section class="section section-padding-medium">
+    <div class="content is-medium">
+      {% if problem_basics["exists"] %}
+
+      <p>
+        <a href="/problem/{{ course_basics["id"] }}/{{ assignment_basics["id"] }}/{{ problem_basics["id"] }}">Back to Problem</a>
+      </p>
+        <h4>Answer Code:</h4>
+
+          <p>
+            <pre>
+              {{ problem_details["answer_code"]}}
+            </pre>
+          </p>
+
+          {% if len(problem_details["answer_description"]) > 0 %}
+            <h4>View the Explanation Here:</h4>
```

Comment:
Instead of saying "View the Explanation Here:", please say "Instructor's comments".